diff --git a/configure.ac b/configure.ac
index 9f97dd2a7..f32141dbe 100644
--- a/configure.ac
+++ b/configure.ac
@@ -358,10 +358,7 @@ AC_CONFIG_FILES([
 AC_CONFIG_FILES([
                  ucx.spec
                  ucx.pc
                  contrib/rpmdef.sh
-                 debian/rules
-                 debian/control
-                 debian/changelog
                  src/ucp/Makefile
                  src/ucp/api/ucp_version.h
                  src/ucp/core/ucp_version.c
