diff --git a/configure.ac b/configure.ac
index 9f97dd2a7..f32141dbe 100644
--- a/configure.ac
+++ b/configure.ac
@@ -360,9 +360,6 @@ AC_CONFIG_FILES([
                  ucx.pc
                  contrib/rpmdef.sh
                  contrib/ucx
-                 debian/rules
-                 debian/control
-                 debian/changelog
                  src/ucp/Makefile
                  src/ucp/api/ucp_version.h
                  src/ucp/core/ucp_version.c
