diff --git a/configure.ac b/configure.ac
index 41c65d047..b1b1f0ca4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -367,9 +367,6 @@ AC_CONFIG_FILES([
                  ucx.spec
                  ucx.pc
                  contrib/rpmdef.sh
-                 debian/rules
-                 debian/control
-                 debian/changelog
                  src/ucp/Makefile
                  src/ucp/api/ucp_version.h
                  src/ucp/core/ucp_version.c
